code/game/objects/items/stacks/cable_coil.dm

```diff
@@ -20,9 +20,9 @@
 	attack_verb = list("whipped", "lashed", "disciplined", "flogged")
 	stack_id = "cable coil"
 
-	suicide_act(mob/user)
-		viewers(user) << "<span class='warning'><b>[user] is strangling \himself with the [src.name]! It looks like \he's trying to commit suicide.</b></span>"
-		return(OXYLOSS)
+/obj/item/stack/cable_coil/suicide_act(mob/user)
+	user.visible_message("<span class='warning'><b>[user] is strangling [p_them()]self with the [src.name]! It looks like [user.p_theyre()] trying to commit suicide.</b></span>")
```

Comment:
class='danger'

Reply:
All suicide act messages are class='warning' here afaik. Should be just class='rose' since it's drama at most.